java/src/main/java/ai/onnxruntime/OnnxTensor.java

```diff
@@ -314,8 +315,9 @@ public static OnnxTensor createTensor(OrtEnvironment env, Object data) throws Or
   }
 
   /**
-   * Create a Tensor from a Java primitive or String multidimensional array. The shape is inferred
-   * from the array using reflection.
+   * Create a Tensor from a Java primitive, String, primitive multidimensional array or String
+   * multidimensional array. The shape is inferred from the object using reflection. The default
+   * allocator is used.
```

Comment:
Has an argument allocator

Reply:
Ooops, I copied that over from the other one. I'll fix it.

Reply:
Fixed.